pkg/healthz/healthz_test.go

```diff
@@ -339,6 +335,11 @@ func (o *fakeResourceLock) Get(ctx context.Context) (*resourcelock.LeaderElectio
 	if err != nil {
 		return nil, nil, err
 	}
+	if o.record.HolderIdentity == remoteIdentity {
+		// Always return unique leader election record bytes,
+		// to simulate an always updated remote leader election record.
+		lerByte = []byte(uuid.NewUUID())
```

Comment:
@wallrj I don't understand this, first lerByte is a json blob and now it is overwritten with a UUID value?

Reply:
https://github.com/cert-manager/cert-manager/pull/6022#discussion_r1186149292 explains the code